DP02_13a_Image_Cutout_SciDemo.ipynb

```diff
@@ -0,0 +1,1286 @@
+{
```

Comment:
<p>The plots in this section and the one before are a little hard to absorb, because they scroll off the screen, making it hard to see more than one at a time. Maybe it would be good to make them smaller so more than one fits on the screen? (Or make a multipanel plot?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>Ok I opted to just shrink each one and consolidate the info in text inside the figure, rather than title. Hopefully this improves clarity. </p>